system-tests/projects/vite3.0.2-react/package.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Above file has the Vite 3 test:

https://github.com/cypress-io/cypress/pull/22915/files#diff-e6bb614dd5b32e07eb125231dc627f64eb0e9a36b6c39cc7b6d23e802b9404b1